crates/holochain_cascade/src/authority/get_agent_activity_query/must_get_agent_activity.rs

```diff
@@ -83,32 +83,48 @@ pub fn filter_then_check(
     }
 }
 
-/// Find the filters sequence bounds.
+/// Find the filter's sequence bounds.
 fn find_bounds(
     txn: &Transaction,
     scratch: Option<&Scratch>,
     author: &AgentPubKey,
     filter: ChainFilter,
 ) -> StateQueryResult<Sequences> {
-    let mut statement = txn.prepare(ACTION_HASH_TO_SEQ)?;
+    let mut hash_statement = txn.prepare(ACTION_HASH_TO_SEQ)?;
```

Comment:
`hash` is a verb, this reads more like a function name. Perhaps `by_hash_stmt` or `by_hash_statement` or even `hash_to_seq_statement`?

Reply:
Changed it to `by_hash_statement`